gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/cockpit/command/handler/EnvironmentCommandHandler.java

```diff
@@ -63,7 +65,7 @@ public Single<EnvironmentReply> handle(EnvironmentCommand command) {
 
             final EnvironmentEntity environment = environmentService.createOrUpdate(
                 environmentPayload.getOrganizationId(),
```

Comment:
I think we should use the organizationId of the existing environment if it exists 🤔